TaskHouseUnitTests/WorkersControllerUnitTests.cs

```diff
@@ -0,0 +1,192 @@
+        {
+            Worker worker = null;
+            int id = 2600;
+
+            var result = await controller.Update(id, worker);
+
+            Assert.IsType<BadRequestResult>(result);
+        }
+
+        [Fact]
+        public async void WorkerController_Update_ReturnsBadRequestResult_WhenIdIsInvalid()
+        {
+            Worker TestWorker = new Worker()
+            {
+                Id = 1,
+                Username = "Tusername2",
+                Password = "+z490sXHo5u0qsSaxbBqEk9KsJtGqNhD8I8mVBdDJls=", //1234
+                Email = "test@test.com",
+                FirstName = "Bob7",
+                LastName = "Bobsen6",
+                Salt = "upYKQSsrlub5JAID61/6pA=="
+            };
+
+            int id = 10000000;
+
+            var result = await controller.Update(id, TestWorker);
+
+            Assert.IsType<BadRequestResult>(result);
+        }
+
+        [Fact]
+        public async void WorkerController_Delete_ReturnsNoContentResult_WhenIdIsValid()
+        {
+            int Id = 1;
+            var result = await controller.Delete(Id);
+
+            Assert.IsType<NoContentResult>(result);
+        }
+
+        [Fact]
+        public async void WorkerController_Update_ReturnsBadRequestResult_WhenWorkerIsNull()
+        {
+            Worker worker = null;
+            int id = 1;
+
+            var result = await controller.Update(id, worker);
+
+            Assert.IsType<BadRequestResult>(result);
+        }
+
+        [Fact]
+        public async void WorkerController_Delete_ActuallyDeletes_WhenIdIsValid()
+        {
+            var Id = 1;
+
+            await controller.Delete(Id);
+            var result = await controller.Get(Id);
+
+            Assert.IsType<NotFoundResult>(result);
+        }
+
+        [Fact]
+        public async void WorkerController_Delete_ReturnsNotFoundResult_WhenIdIsInvalid()
+        {
+            int id = 2500;
+      
+            var result = await controller.Delete(id);
+
+            Assert.IsType<NotFoundResult>(result);
+        }
+
+        [Fact]
+        public async void WorkerController_Update_ReturnsObjectResult_withValidIdAndValidWorker()
+        {
+            Worker worker1 = new Worker()
+            {
+                Id = 10,
+                Username = "Tusername",
+                Password = "+z490sXHo5u0qsSaxbBqEk9KsJtGqNhD8I8mVBdDJls=", //1234
+                Email = "test@test.com",
+                FirstName = "Bob7",
+                LastName = "Bobsen6",
+                Salt = "upYKQSsrlub5JAID61/6pA=="
+            };
+            Worker worker2 = new Worker()
+            {
+                Id = 10,
+                Username = "Tusername234",
+                Password = "+z490sXHo5u0qsSaxbBqEk9KsJtGqNhD8I8mVBdDJls=", //1234
+                Email = "test@test.com",
+                FirstName = "Bob7",
+                LastName = "Bobsen6",
+                Salt = "upYKQSsrlub5JAID61/6pA=="
+            };
```

Comment:
Use the workers from your FakeRepo. It will simplify your tests and eliminate one of the worker instances in the test. Look at [SkillControllerTest](https://github.com/taskhouse/server/blob/master/TaskHouseUnitTests/SkillsControllerUnitTests.cs#L107).